app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/text_input.scala.html

```diff
@@ -49,7 +50,7 @@
   @if(labelAsHeading) {
     <h1><label for="@{fieldId}" class="heading-xlarge">@{label}</label></h1>
   } else {
-    <label for="@{fieldId}">@{label}</label>
+    <label for="@{fieldId}" class="@{labelClass.getOrElse("")}">@{label}</label>
```

Comment:
would you want `class=` a valid thing in the html if `labelClass` is not defined?

Reply:
yes but scrappy - added in the default label class anyway which conveniently stops it ever being empty